admin_guide/managing_networking.adoc

```diff
@@ -547,7 +547,7 @@ moves it into the pod's network project before starting the *egress-router*
 container. Preserve the quotation marks around `"true"`. Omitting them results
 in errors. To create the Macvlan interface on a network interface other than the primary one, set the annotation value to the name of that interface. For example, `eth1`.
 <2> IP address from the physical network that the node is on and is
-reserved by the cluster administrator for use by this pod.
+reserved by the cluster administrator for use by this pod. Optionally, you can specify a CIDR so that the egress router can access the subnet: `192.168.12.99/24`.
```

Comment:
I think it would probably be better to change the examples to use the CIDR form, because it's not really that hard for the user to do, and it results in fewer surprises.

So change the `EGRESS_SOURCE` value above in the yaml to `192.168.12.99/24`, and then change the footnote here to something like:

> IP address from the physical network that the node is on and is reserved by the cluster administrator for use by this pod. For best results, the IP address should include the subnet length (eg, the `/24` here) so that a proper route to the local subnet can be set up. (If you do not include a subnet length then the egress router will not be able to access other hosts on its subnet, besides the `EGRESS_GATEWAY` host.)

Then in the following examples that don't currently have `EGRESS_SOURCE` footnotes, just add the `/24`, and in the ones that do have `EGRESS_SOURCE` footnotes, copy the same text as you put here